frontend/assets/images/icons/empty-folder-svgrepo-com.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+	 viewBox="0 0 56.724 56.724" style="enable-background:new 0 0 56.724 56.724;" xml:space="preserve">
+<g>
+	<path style="fill:#010002;" d="M7.487,46.862c1.111,1.467,2.865,2.275,4.938,2.275h30.598c3.984,0,7.934-3.009,8.991-6.849

```

Comment:
Can we set the fill colour to `#434b53` ? Most of the other icons are using this color.

Reply:
@Navaneeth-pk Desired changes done